docs/tutorials/python/submission.md

```diff
@@ -0,0 +1,131 @@
+# Submissions, SubmissionStatuses, SubmissionBundles
```

Comment:
I am not seeing this in the index page of tutorial: 

<img width="938" height="800" alt="Screenshot 2025-12-02 at 1 57 43 PM" src="https://github.com/user-attachments/assets/258eefb3-8884-470f-9f9f-9209f00b5149" />